pycbc/__init__.py

```diff
@@ -45,7 +46,21 @@
 __version__ = pycbc_version
 
 
-def init_logging(verbose=False, format='%(asctime)s %(message)s'):
+_default_format = '%(asctime)s %(message)s'
```

Comment:
I actually suggest keeping the default string in the function signature, because that makes the default show up explicitly in the documentation. I think that would be lost if we do this change.